### DISCUSSION TITLE
Render a pdf program/agenda

### DISCUSSION THREAD

**Discussion by rixx** [pretalx/pretalx#1070] [Ideas]
This should include dates/times, but should emphasize text and images, more like a normal conference booklet.

**Comment by oxinabox** [comment]
For interest this is how I generate a program book.
It is a script that generates latex, the latex can then be compile directly (as in the example below),
or `include`ed into a larged document with e.g. a cover, a map, a letter from the chair etc)
In general this is probably not great for most users, but it is for certain academic user.

https://github.com/oxinabox/DeadConPrinter/blob/master/quick_con_program_book/make_con_book_pretalx_2019_juliacon.ipynb

(The way it is currently working for Pretalx is via a sperate script that reads sthe API and writes a CSV but that is not nesc, it was just because I had that generated CSV handy. For Grenadine I had  it reading the API directly.)


Example: https://github.com/oxinabox/DeadConPrinter/blob/master/quick_con_program_book/example_output/example_pretty/sc2015_d02.pdf

Anyway, I thought you might find it interesting.

**Comment by rixx** [comment]
Thank you for sharing, this is really cool!

**Comment by astro** [comment]
I use XSL:FO to render schedule XML into an agenda-style PDF: https://gitea.c3d2.de/c3d2/c3d2-web/src/branch/master/xslfo/datenspuren/schedule
